src/components/Dashboard/TodoList.vue

```diff
@@ -858,14 +858,19 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     return !!item.affiliationInvitationDetails
   }
 
-  authorizeAffiliationInvitation (isAuthorized, affiliationInvitationTodo): void {
+  authorizeAffiliationInvitation (isAuthorized, affiliationInvitationTodo: TodoItemIF): void {
+    const currentItems:Array<TodoItemIF> = this.todoItems
```

Comment:
Why are you creating a pointer (another reference) to `this.todoItems` ?

By the way, since this.todoItems has a type, you don't need to type currentItems.

Reply:
Updated a code to be a bit more readable

Reply:
Yes, it's more readable now. Thanks!